packages/server-side-graphql-client/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@keystonejs/server-side-graphql-client",
+  "description": "A library for running server-side graphQL queries and mutations against in Keystone.",
```

Comment:
"against in Keystone" ?

Reply:
Almost makes sense if you squint... but agree not a top-shelf sentence.